src/pkg/scan/export/manager.go

```diff
@@ -0,0 +1,230 @@
+package export
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	beego_orm "github.com/beego/beego/orm"
+	"github.com/goharbor/harbor/src/jobservice/logger"
+	"github.com/goharbor/harbor/src/lib/orm"
+	q2 "github.com/goharbor/harbor/src/lib/q"
+	"strconv"
+	"strings"
+)
+
+const (
+	VulnScanReportView  = "vuln_scan_report"
+	VulnScanReportQuery = `select row_number() over() as result_row_id, project.project_id as project_id, project."name" as project_name, harbor_user.user_id as user_id, harbor_user.username as project_owner, repository.repository_id, repository.name as repository_name, artifact.digest as artifact_digest, scanner_registration.id as scanner_id, scanner_registration."name" as scanner_name, vulnerability_record.cve_id, vulnerability_record.package, vulnerability_record.severity,
```

Comment:
as for the CSV column, 

- the repository name is enough.

- no need the project owner.

Reply:
The reason that the project owner was included was because when an admin runs a report, they should be able to immediately filter the obtained CSV based on certain fields of which project owner would be an important one. For e.g. the current implementation facilitates a workflow where the admin downloads the CSV for all projects and then performs targeted analysis in Excel for  specific set of project owners.

Reply:
As discussed in a meeting, I will be currently keeping the project owner column as is and then we can decide the exact columns to be placed in the CSV based on the performance benchmarks.